packages/front-end/components/Layout/Layout.module.scss

```diff
@@ -15,7 +15,7 @@ $topbarheight: 60px;
   color: #fff;
   overflow: hidden;
   overflow-y: auto;
-  font-family: Poppins, sans-serif;
+  font-family: Inter, sans-serif;
```

Comment:
If we're no longer using Poppins anywhere, can we remove the import in `global.scss`?

Reply:
i think we can remove poppins i wanted to double check that the Kari
but yes we can remove it if we are not going to use it.

Reply:
it can be removed :)